api/types/backend/backend.go

```diff
@@ -67,3 +67,11 @@ type ExecProcessConfig struct {
 	Privileged *bool    `json:"privileged,omitempty"`
 	User       string   `json:"user,omitempty"`
 }
+
+// ContainerCommitConfig is a wrapper around
+// types.ContainerCommitConfig that also
+// transports configuration changes for a container.
+type ContainerCommitConfig struct {
+	types.ContainerCommitConfig
+	Changes []string
```

Comment:
As I said in the description:

> I know the backend.ContainerCommitConfig struct could be merged into the other
struct, but I rather focus in removing this dependency now and cleaning up
the struct later to move forward with the API cleaning.